borg/archive.py

```diff
@@ -589,7 +572,10 @@ def process_file(self, path, st, cache, ignore_inode=False):
                 status = 'U'  # regular file, unchanged
         else:
             status = 'A'  # regular file, added
-        item = {b'path': safe_path}
+        item = {
+            b'path': safe_path,
+            b'link': st.st_nlink > 1,
```

Comment:
hm, no. meaning of "link" is rather "this item has the data for a set of hardlinks".